src/proxy-ui-api/frontend/src/store/modules/certificateSignRequest.ts

```diff
@@ -103,6 +106,10 @@ export const csrGetters: GetterTree<CsrState, RootState> = {
     return state.keyLabel;
   },
   isUsageReadOnly(state): boolean {
+    // if creating CSR for a hardware token, only sign CSRs can be created
+    if (state.tokenType == TokenType.HARDWARE) {
```

Comment:
In this case it doesn't matter but it's a good practise to always use three equal signs when doing a comparison in javascript.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality

For example. 

1 == '1' // true 
1 === '1' // false

Reply:
fixed